flink-libraries/flink-python/src/test/python/org/apache/flink/python/api/ut_pyflink.py

```diff
@@ -54,29 +50,14 @@ def parse_args():
 	return local_options
 
 
-# The python path argument should be present in the run mode when the python file/plane is run in a different
-# location. The assumption is that the path provided is absolute path
-def fix_python_path(python_path):
-	if python_path is not None:
-		path_list = options.python_path.split(":")
-		for p in path_list:
-			sys.path.append(p)
-
-
 # This code should run before any other piece of code using the flink python stuff
 options = parse_args()
 if options.verbose:
 	print("In first main section")
-fix_python_path(options.python_path)
 
 
-# Flink python testing imports
-if options.mvn_mode:
```

Comment:
the mvn_mode parameter was never supplied when retrieving the method names causing this import to fail.